### DISCUSSION TITLE
自定义行头，列头，当点击行头和列头的时候，能不选中当前行和列数据单元格吗

### DISCUSSION THREAD

**Discussion by jiudehuiyi** [antvis/S2#2500] [Q&A]
<img width="669" alt="image" src="https://github.com/antvis/S2/assets/47971539/ad826b3e-171d-4708-b4f0-4dea9b8934d5">
<img width="720" alt="image" src="https://github.com/antvis/S2/assets/47971539/18e13ece-9905-411f-a4a4-2cf77e4d1c83">

**Comment by lijinke666** [comment]
不能
